core/src/test/java/org/hibernate/ogm/backendtck/associations/collection/manytomany/ManyToManyExtraTest.java

```diff
@@ -76,14 +73,18 @@ private int expectedAssociationNumber() {
 			// In Neo4j relationships are bidirectional
 			return 1;
 		}
+		else if ( TestHelper.getCurrentDialectType().equals( GridDialectType.ORIENTDB ) ) {
+			// In OrientDB ManyToMany relationships is like in RDBMS
+			return 3;
+		}
 		else {
 			return 2;
 		}
 	}
 
 	@Override
 	protected Class<?>[] getAnnotatedClasses() {
-		return new Class<?>[] {
+		return new Class<?>[]{
```

Comment:
Please, make sure not to change the format of the source code if it's not needed. There are already plenty of changes in this PR